middleware/redirects/redirects.go

```diff
@@ -16,6 +19,16 @@ var redirects = make(map[string]string)
 var PanicOnInitError = true
 
 func Init(asset func(name string) ([]byte, error)) {
```

Comment:
I checked that `Init` func is only called from main, and main also loads the config. If we need to access the config from here, we could accept it as a paramter, so that we don't need to load it again (or if it was just added to get the assetsDir, then maybe we can just remove this bloc of code?)

Reply:
Yes this is some code that ended up not being needed, will remove